### PR TITLE
Refactor country lists to ensure better separation 

### PR DESCRIPTION
**Pull request by paulbrown1982**
## What are you doing in this PR?
Refactoring country lists to ensure separation between universal list, GW billing and delivery lists, and a specific list for SEPA-eligible countries. Also I added better type protection `IsoCountry` to the lists to better ensure compile-time safety.

## Testing
- [x] Tested on CODE.

### CODE REVIEW

File: yarn.lock
Comment (undergroundquizscene) [inline yarn.lock:1]: Should this file be here?

File: support-frontend/assets/helpers/internationalisation/billableCountries.ts [hunk 1]
```diff
@@ -0,0 +1,240 @@
+const billableCountries: Record<string, string> = {
```
Comment (undergroundquizscene) [inline support-frontend/assets/helpers/internationalisation/billableCountries.ts:1]: Is it worth using `IsoCountry` (from internationalisation/country) for this and the other similar types?

```suggestion
const billableCountries: Record<IsoCountry, string> = {
```

**Review by undergroundquizscene — APPROVED** [review]

File: support-frontend/assets/helpers/internationalisation/billableCountries.ts (hunk 1 above)
Comment (paulbrown1982) [reply support-frontend/assets/helpers/internationalisation/billableCountries.ts]: Addressed.

**Comment by paulbrown1982** [comment]
Superceded by https://github.com/guardian/support-frontend/pull/4996

**Pull request merged**